deploy_configs/11155111.json

```diff
@@ -4,6 +4,7 @@
   "committeeMembers": ["0x68b43fd906c0b8f024a18c56e06744f7c6157c65", "0xacaef39832cb995c4e049437a3e2ec6a7bad1ab5", "0x8061f127910e8ef56f16a2c411220bad25d61444", "0x508f3f1ff45f4ca3d8e86cdcc91445f00acc59fc"],
   "sourceChainId": 11,
   "supportedTokens": ["0x0112D7B36726B3077b72DDb457A9f9c94D9cd71c", "0xfFf9976782d46CC05630D1f6eBAb18b2324d6B14", "0x80bF6fb931C8eB99Ab32aeD543ACCFd168fd2a47"],
+  "tokenSuiDecimals": [9,8,8,6,6],
```

Comment:
is there a middle ground where we keep them in a mapping but the initial values are hardcoded in initializer? I sort of think this level of configurability is unnecessary and make it a bit more error-prone. But perhaps i'm missing something!

Reply:
I see the point about this being more error prone... I will say enabling configurability like this enables us to write tests for the decimal conversion logic more easily. I'm not sure how we'd write tests for the decimal conversion functions with:
```require(ethDecimal > suiDecimal, "BridgeTokens: Invalid Sui decimal");```
if the suiDecimal values were constant.

Reply:
If we added the values in the initializer we'd need to require 4 addresses are provided (which isn't a huge deal) so that there is an address to mach each suiDecimal value.

Reply:
How's this? 
```
    /// @notice Constructor function for the BridgeTokens contract.
    /// @dev the provided arrays must have the same length.
    /// @param _supportedTokens The addresses of the supported tokens.
    constructor(address[] memory _supportedTokens) Ownable(msg.sender) {
        require(_supportedTokens.length == 4, "BridgeTokens: Invalid supported token addresses");

        uint8[] memory _suiDecimals = new uint8[](5);
        _suiDecimals[0] = 9; // SUI
        _suiDecimals[1] = 8; // wBTC
        _suiDecimals[2] = 8; // wETH
        _suiDecimals[3] = 6; // USDC
        _suiDecimals[4] = 6; // USDT

        // Add SUI as the first supported token
        supportedTokens[0] = Token(address(0), _suiDecimals[0]);

        for (uint8 i; i < _supportedTokens.length; i++) {
            supportedTokens[i + 1] = Token(_supportedTokens[i], _suiDecimals[i + 1]);
        }
    }
```

Reply:
lgtm!